site/docs/voice/webhooks/mediaStreamRejected.mdx

```diff
@@ -0,0 +1,81 @@
+---
+id: mediaStreamRejected
+title: Media Stream Rejected
+slug: /voice/webhooks/mediaStreamRejected
+description: A general overview of Bandwidth's Media Stream Rejected Webhook
+keywords:
+  - bandwidth
+  - voice
+  - webhook
+  - streaming
+  - rejected
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+This event is sent to the url specified when sending a [`<StartStream>`][1] verb. This event will be sent if the stream requested in the `<StartStream>` verb was not started.
```

Comment:
```suggestion
This event may be sent to the url specified when sending a [`<StartStream>`][1] verb. This event will be sent if the stream requested in the `<StartStream>` verb was not started.
```
Incredibly nitpicky but since it's one of three webhooks that can possibly be sent, I think saying `is sent` could be confusing (same for the other two). Feel free to ignore this though